Port changes for installing GPU driver from 2022.3 branch (#15408)
openvinotoolkit/openvino: docs/install_guides/configurations-for-intel-gpu.md

@endsphinxdirective

> In case if you are intended to use OpenVINO GPU plugin and offload network inference to Intel® graphics processor, the Intel Graphics Driver should be properly configured on your system.

Reviewer: ```suggestion
If you are intended to use OpenVINO™ GPU plugin and offload inference to Intel® Processor Graphics (GPU), Intel® Graphics Driver must be properly configured on your system.
```

Author: This is port of changes from PR #15160 that's already approved by docs team.
Could you please clarify why we have such difference

> If it is already installed, and you want to keep it, you can skip the installation steps.

Reviewer: ```suggestion
If you have already installed Intel® Graphics Driver and would like to keep it, you can skip the installation steps below.
```

## Linux

> To install the latest available **Intel® Graphics Compute Runtime for OpenCL™** for your OS, see the [Install Guides](https://github.com/intel/compute-runtime/releases/latest).

Reviewer: ```suggestion
To install the latest available **Intel® Graphics Compute Runtime for OpenCL™** for your operating system, see the [Install Guides for Intel® Graphics Compute Runtime for oneAPI Level Zero and OpenCL™ Driver](https://github.com/intel/compute-runtime/releases/latest).
```

> > **NOTE**: If you use RedHat 8 OS please install OpenCL library as prerequisite via following command line:  

Reviewer: ```suggestion
> **NOTE**: If you are using RedHat 8, use the following command to install the OpenCL library as a prerequisite:  
```

> ```sh rpm -ivh http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/ocl-icd-2.2.12-1.el8.x86_64.rpm```

> > **NOTE**: For instructions specific to discrete graphics platforms, refer to [the dgpu guide](https://dgpu-docs.intel.com/installation-guides/index.html) (Intel® Arc™ A-Series Graphics, Intel® Data Center GPU Flex Series, Intel® Data Center GPU MAX Series, Intel® processor graphics Gen12, and Intel® Iris Xe MAX codename DG1).

Reviewer: ```suggestion
For instructions for discrete graphics platforms, refer to [the dGPU guides](https://dgpu-docs.intel.com/installation-guides/index.html), including Intel® Arc™ A-Series Graphics, Intel® Data Center GPU Flex Series, Intel® Data Center GPU MAX Series, Intel® processor graphics Gen12, and Intel® Iris Xe MAX codename DG1.
```

> You may consider installing one of the earlier versions of the driver, based on your particular setup needs.

Reviewer: ```suggestion
You can also install one of the earlier versions of the driver based on your particular setup needs.
```

> It is recommended that you refer to the [Intel® Graphics Compute Runtime Github page](https://github.com/intel/compute-runtime/) for instructions and recommendations on GPU driver installation specific to particular releases, including the list of supported hardware platforms.

Reviewer: ```suggestion
For instructions and recommendations on the installation of particular releases of the GPU driver, as well as the list of supported hardware platforms, see the [Intel® Graphics Compute Runtime GitHub page](https://github.com/intel/compute-runtime/).
```

## Windows

> To install the Intel Graphics Driver for Windows on your hardware, please proceed with the [instruction](https://www.intel.com/content/www/us/en/support/articles/000005629/graphics.html). 

Reviewer: ```suggestion
To install the Intel® Graphics Driver for Windows on your hardware, proceed with [this article on the Support site](https://www.intel.com/content/www/us/en/support/articles/000005629/graphics.html). 
```

To check if you have this driver installed:

1. Type **device manager** in your **Search Windows** box and press Enter. The **Device Manager** opens.
2. Click the drop-down arrow to view the **Display adapters**. You can see the adapter that is installed in your computer:  

> ![](../img/DeviceManager.PNG)

Reviewer: ```suggestion
   ![](../img/DeviceManager.PNG)
```

3. Right-click the adapter name and select **Properties**.
4. Click the **Driver** tab to see the driver version.  

> ![](../img/DeviceDriverVersion.PNG)

Reviewer: ```suggestion
   ![](../img/DeviceDriverVersion.PNG)
```

You are done updating your device driver and are ready to use your GPU.

> ## Additional info

Reviewer: ```suggestion
## Additional Information
```

> In the internal OpenVINO validation the following versions of Intel Graphics Driver were used:

Reviewer: ```suggestion
For your reference, the following versions of Intel® Graphics Driver are validated internally:
```

> Operation System | Driver version

Reviewer: ```suggestion
Operating System | Driver version
```
